backend/config/settings/base.py

```diff
@@ -59,6 +62,13 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+AUTHENTICATION_BACKENDS = [
+    "social_core.backends.open_id.OpenIdAuth",
```

Comment:
Wouldn't OAuth2 be enough? Who chooses which one to use if there are many of supported protocols?